common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -97,6 +97,15 @@ class ProctoringFields(object):
         scope=Scope.settings,
     )
 
+    hide_after_due = Boolean(
+        display_name=_("Hide Exam Results After Due Date"),
+        help=_(
+            "This setting overrides the default behavior of showing exam results after the due date has passed."
```

Comment:
@efischer19 The instances I'm seeing of timed exam etc. with initial caps look to me like titles or labels, so initial caps are fine in those cases. But in a sentence, it would be best to lowercase "timed exams", so let's do that here. I looked on the sandbox incl. in Advanced Settings but am not sure where this text shows up?